test/conftest.py

```diff
@@ -136,6 +136,9 @@ def pg(tmp_path_factory, cert_dir):
     pg.sql(f"create user longpass with password '{LONG_PASSWORD}';")
     pg.sql("create user stats password 'stats';")
     pg.sql("grant all on schema public to public", dbname="p0")
+    pg.sql("create table t(i int)", dbname="p0")
+    pg.sql("grant all on table t to public", dbname="p0")
+
```

Comment:
Rename table "t" to "test_copy". It seems clear that it is used by `test_copy.py`.